### ISSUE TITLE
[SVG renderer] Wrong Axis Popover display when cursor over a chart

### ISSUE THREAD

**Issue by Paul-Kijtapart** [apache/incubator-echarts#7946]
### One-line summary [问题简述]
Wrong Axis Popover display when cursor over a chart

### Version & Environment [版本及环境]
+ ECharts version [ECharts 版本]: 4.0.4
+ Browser version [浏览器类型和版本]: Chrome
+ OS Version [操作系统类型和版本]: Window 10

### Expected behaviour [期望结果]
When cursor is hover over a Line chart, user should see popover box over x-axis and y-axis at the coordinate of the hover.
When renderer (used in initOpts) is set as "svg", the pop-up boxes should have the same style as that of "canvas".

### ECharts option [ECharts配置项]

```javascript
option = {
  // Chart is displayed perfectly fine with renderer as "canvas". 
 // the issue occurs when I set renderer as "svg"
}

```

### Other comments [其他信息]

https://i.imgur.com/aY5U77m.png

**Comment by 100pah** [comment]
The same as #7947

**Comment by Paul-Kijtapart** [comment]
Solved in #7947
